components/renku_data_services/project/orm.py

```diff
@@ -53,8 +53,28 @@ def dump(self) -> models.Project:
             name=self.name,
             slug=self.slug,
             visibility=self.visibility,
-            created_by=models.User(id=self.created_by_id),
+            created_by=models.Member(id=self.created_by_id),
             creation_date=self.creation_date,
             repositories=self.repositories,
             description=self.description,
         )
+
+
+class ProjectMemberORM(BaseORM):
```

Comment:
What is the purpose of this table? The authz service and database already contains pretty much the same information. 

I was hoping that the authz service and its tables would be the only source of truth for authorization data and membership.

Reply:
See `components/renku_data_services/authz/orm.py`

Reply:
As agreed, this will be done in a future PR.